cloud/scaling.rst

```diff
@@ -0,0 +1,42 @@
+Scaling
+#######
+
+Citus Cloud provides self-service scaling to deal with increased load. The web interface makes it easy to either add new worker nodes or increase existing nodes' memory capacity.
+
+For most cases either approach to scaling is fine and will improve performance. However there are times you may want to choose one over the other. If the cluster is reaching its disk limit then adding more nodes is the best choice. Alternately, if there is still a lot of headroom on disk but performance is suffering, then scaling node RAM is the way to go.
+
+Both adjustments are available in the formation configuration panel of the settings tab:
+
+.. image:: ../images/cloud-formation-configuration.png
+
+Clicking the node change link provides two sliders:
+
+.. image:: ../images/cloud-nodes-slider.png
+
+The first slider, **count**, scales out the cluster by adding new nodes. The second, **RAM**, scales it up by changing the amount of RAM on existing nodes.
+
+After you adjust the sliders and accept the changes, Citus Cloud begins applying the changes subject to the user-specified maintenance window:
+
+.. image:: ../images/cloud-maintenance-window.png
```

Comment:
I had some questions here:

- Does the maintenance window apply for scaling out as well? I can understand for scaling up, since it requires a switchover, but scaling out shouldn't require any downtime? 

- What considerations should we communicate to the user about the maintenance window? We discuss a bit below about time it takes to synchronize (for scaling up), but how much downtime should they expect? Is that downtime only during the switchover?

- Is there subsequent information we want to communicate? Do I get emails when its done? Does it show in the dashboard as "configuring" or something?